application-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -478,69 +560,4 @@ XWiki.widgets.StepPopup = Class.create(XWiki.widgets.ModalPopup, {
       <use>onDemand</use>
     </property>
   </object>
-  <content>{{velocity}}
-{{html wiki=true}}
-#set($isNewStep = true)
-#set($bSTDocRef = $request.BSTDocRef)
-#set($stepID = $request.stepID)
-#set($nbSteps = 0)
-#if ($bSTDocRef &amp;&amp; $bSTDocRef != "" &amp;&amp; $xwiki.exists($bSTDocRef))
-   #set($bSTDoc = $xwiki.getDocument($bSTDocRef))
-   #set($nbSteps = $bSTDoc.getObjects("TourCode.StepClass").size())
-   #if($stepID &amp;&amp; $stepID != "")
-      #set($stepObj = $bSTDoc.getObject("TourCode.StepClass", $mathtool.toInteger($stepID)))
-      #if($stepObj)
-         #set($isNewStep = false)
-      #end
-   #end
-#end
-#if ($isNewStep)
-   #set($vdoc = $xwiki.getDocument("TourCode.StepTemplate"))
-   #set($vobj = $vdoc.getObject('TourCode.StepClass'))
-#else
-   #set($vdoc = $bSTDoc)
-   #set($vobj = $stepObj)
-#end
-#set($class = $vobj.xWikiClass)
-#if ($isNewStep &amp;&amp; $isNewStep != "")
-   &lt;h3&gt;$services.localization.render('tour.steps.new_step')&lt;/h3&gt;
-#else
-   &lt;h3&gt;$services.localization.render('tour.steps.edit_step')&lt;/h3&gt;
-#end
-&lt;div id="stepForm" class="xform"&gt;
- #foreach($prop in $class.properties)
-    #if($prop.getName().equals("order"))
-       ; &lt;label&gt;$services.localization.render("${class.name}_${prop.name}")&lt;/label&gt;
-       &lt;dd&gt;
-       &lt;select id="stepOrderSelect"&gt;
-          #foreach($i in [0..$mathtool.sub($nbSteps, 1)])
-             &lt;option value="$i" #if ($vobj.getProperty("order").value == $i) selected #end &gt;$mathtool.add($i, 1)&lt;/option&gt;
-          #end
-          #if ($isNewStep)
-             &lt;option value="$nbSteps" selected &gt;$mathtool.add($nbSteps, 1)&lt;/option&gt;
-          #end
-       &lt;/select&gt;
-       &lt;/dd&gt;
-    #else
-       ; &lt;label for="TourCode.StepClass_0_${prop.name}"&gt;$services.localization.render("${class.name}_${prop.name}")&lt;/label&gt;
-       : $vobj.get($prop.getName())
-    #end
- #end
-
-#if ($context.action == "edit")
-   #if($request.BSTDocRef &amp;&amp; $request.BSTDocRef != "")
-      &lt;input type="hidden" value="$request.BSTDocRef" id="BSTDocRef"/&gt;
-   #end
-   #if ($request.stepID)
-      &lt;input type="hidden" value="$request.stepID" id="stepID"/&gt;
-   #end
-   #if ($isNewStep &amp;&amp; $isNewStep != "")
-      &lt;input type="button" value="$services.localization.render('tour.steps.add_step')" class="btn btn-primary" id="saveStepBtn"/&gt;
-   #else
-      &lt;input type="button" value="$services.localization.render('tour.steps.edit_step')" class="btn btn-primary" id="saveStepBtn"/&gt;
-   #end
-#end
-&lt;/div&gt;
-{{/html}}
-{{/velocity}}</content>
```

Comment:
Unfortunately, this page is unreviewable because you are mixing a XAR format update with code changes. They need to be done separately in order for the commit to be readable.